sdk/tables/azure-data-tables/tests/test_table_client.py

```diff
@@ -433,7 +433,7 @@ def callback(response):
         tables = list(service.list_tables(raw_response_hook=callback))
         self.assertIsInstance(tables, list)
 
-    # @pytest.mark.skip("pending")
+    @pytest.mark.skip("pending")
```

Comment:
Is there a reason this one is skipped?

Reply:
No it is passing fine, I will uncomment it. Good catch!